### PR TITLE
Untangling: Add Advanced Tools -> Logs

### PR DESCRIPTION
**Pull request by fushar**
Fixes:

- https://github.com/Automattic/dotcom-forge/issues/9618

## Proposed Changes

This PR adds the Advanced Tools -> Monitoring subtab under the `untangling/hosting-menu` flag.

image

## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to /sites
2. Click an Atomic site
3. Go to Advanced Tools -> Logs subtab.
4. Verify that this tool is working (check for php and web subtabs).
5. Regression testing:
    - Go to /sites?flags=-untangling/hosting-menu
    - Click an Atomic site
    - Verify that the original Logs tab is still working.

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [x] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/sites/components/site-preview-pane/dotcom-preview-pane.tsx [hunk 1]
```diff
@@ -109,7 +110,7 @@ const DotcomPreviewPane = ( {
 			},
 			{
 				label: __( 'Logs' ),
-				enabled: isActiveAtomicSite,
+				enabled: isActiveAtomicSite && ! config.isEnabled( 'performance-profiler/logged-in' ),
```
Comment (taipeicoder) [inline client/sites/components/site-preview-pane/dotcom-preview-pane.tsx]: @fushar what's the context for this flag? Also, should we check for the `untangling/hosting-menu` flag here?

Comment (fushar) [reply client/sites/components/site-preview-pane/dotcom-preview-pane.tsx]: 🤦 wrong copy-paste, fixed, thanks!

**Comment by taipeicoder** [comment]
I'm getting the site selector when clicking the Web Server tab, is this expected?

https://github.com/user-attachments/assets/c260563f-7a5f-47ca-86ed-bcb6c1194d3f

**Comment by fushar** [comment]
> I'm getting the site selector when clicking the Web Server tab, is this expected?

No, it's not expected; thanks for catching that, fixed!

**Review by taipeicoder — APPROVED** [review]
Looks good! 🚢

**Pull request merged**